src/evalfunc.c

```diff
@@ -2124,6 +2124,8 @@ static funcentry_T global_functions[] =
 			ret_string,	    f_inputsecret},
     {"insert",		2, 3, FEARG_1,	    arg23_insert,
 			ret_first_arg,	    f_insert},
+    {"instanceof",	2, 2, FEARG_1,	    NULL,
```

Comment:
The argument types need to be checked here, so that the types are checked compilation.  Instead of using "NULL",
can you implement a arg2_object_class function?